src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/_help.py

```diff
@@ -648,11 +648,16 @@
   - name: --subnet-prefix
     type: string
     short-summary: The CIDR used on the Subnet into which to deploy the cluster.
+  - name: --customer-admin-group-id
+    type: string
+    short-summary: The ID of an Azure Active Directory Group that memberships will get synced into the OpenShift group "osa-customer-admins". If not specified no cluster admin access will be granted
```

Comment:
Could you change the second sentence here to have a comma and end with a period?
`If not specified, no cluster admin access will be granted.`

Reply:
looks like this was already updated